Riot/Modules/Room/Members/Detail/RoomMemberDetailsViewController.m

```diff
@@ -399,7 +399,14 @@ - (void)updateMemberInfo
         
         self.roomMemberStatusLabel.text = presenceText;
         
-        self.roomMemberAvatarBadgeImageView.image = [EncryptionTrustLevelBadgeImageHelper userBadgeImageFor:self.encryptionTrustLevel];
+        if (self.mxRoom.summary.isEncrypted)
```

Comment:
If `self.encryptionTrustLevel` this is not needed to check that as `EncryptionTrustLevelBadgeImageHelper: userBadgeImageFor:` will return nil for `UserEncryptionTrustLevelNone`

Reply:
You right, updated.